database/rrdset.c

```diff
@@ -466,13 +466,6 @@ void rrdset_delete_custom(RRDSET *st, int db_rotated) {
     }
 
     recursively_delete_dir(st->cache_dir, "left-over chart");
-#ifdef ENABLE_ACLK
-    if ((netdata_cloud_setting) && (db_rotated || RRD_MEMORY_MODE_DBENGINE != st->rrd_memory_mode)) {
```

Comment:
Hello @underhood ,

Due the changes you are doing in this PR, I observed that we are receiving this warning during compilation time: 

```sh
database/rrdset.c:445:43: warning: unused parameter 'db_rotated' [-Wunused-parameter]
  445 | void rrdset_delete_custom(RRDSET *st, int db_rotated) {
         |                                                                    ~~~~^~~~~~~~~~
```

Reply:
fixed in last commit